ably-iosTests/ARTRealtimePresenceTest.m

```diff
@@ -825,7 +826,7 @@ - (void)testWithNoClientIdUpdateLeaveEnterAnotherClient {
             XCTAssertEqualObjects(otherClientId, message.clientId);
             if(messageCount ==0) {
                 XCTAssertEqual(message.action, ARTPresenceEnter);
-                XCTAssertEqualObjects( message.content, nil);
+                XCTAssertEqualObjects(message.content, @"");
```

Comment:
I realise data is an empty string above in line 769, however should we not be testing enter with no data i.e. nil as we have later tests testing the payload data